website/docs/r/edgegateway.html.markdown

```diff
@@ -84,6 +84,12 @@ order) logging. Default `false`.
 * `fw_default_rule_action` (Optional) Default firewall rule (last in the processing order) action.
 One of `accept` or `deny`. Default `deny`.
 
+## Attribute Reference
+
+The following attributes are exported on this resource:
+
+* `default_network_ip` (*v2.6+*) - IP address of edge gateway used for default network
```

Comment:
I'm thinking that it would be useful to show case this new field in one of our examples. It could be here or in some other resource page where EG IP is used.

Reply:
Added two examples in load balancer virtual server and firewall rule